plugins/inputs/radius/radius.go

```diff
@@ -0,0 +1,104 @@
+package radius
+
+import (
+	"context"
+	_ "embed"
+	"net"
+	"sync"
+	"time"
+
+	"layeh.com/radius"
+	"layeh.com/radius/rfc2865"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Radius struct {
+	Servers         []string
+	Username        string
+	Password        string
```

Comment:
If you change those to 
```suggestion
	Username        config.Secret `toml:"username"`
	Password        config.Secret `toml:"password"`
```
you can benefit from [secret-stores](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#secret-store-secrets).